### DISCUSSION TITLE
findRaw $meta mongo how to retrieve score

### DISCUSSION THREAD

**Discussion by MuraraAllan** [prisma/prisma#13472] [Q&A] [answered]
How can I retrieve $meta - textScore for a findRaw query such as   
```
filter: {
      $text: {
        $search: "SPRITE",
      },
    },
``` 
?

Thank you so much

**Comment by MuraraAllan** [comment, marked as answer]
```
const foundProducts = await db.product.findRaw({
    filter: {
      $text: {
        $search: "SPRITE 600 ml",
      },
    },
    options : {
      limit: 15,
      projection: {
        score: { $meta: 'textScore' }
      }
    }
  })
```

**Reply by MuraraAllan** [reply]
https://www.mongodb.com/docs/manual/reference/command/find/#std-label-find-cmd-sort

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
